src/pages/FindAccount/FindAccount.tsx

```diff
@@ -1,13 +1,46 @@
+import { useState } from 'react';
+
+import FindEmailForm from '../../components/FindAccount/FindEmailForm';
+import ResetPasswordForm from '../../components/FindAccount/ResetPasswordForm';
+import GenericModal from '../../components/Modals/GenericModal';
```

Comment:
```suggestion
import FindEmailForm from '@/components/FindAccount/FindEmailForm';
import ResetPasswordForm from '@/components/FindAccount/ResetPasswordForm';
import GenericModal from '@/components/Modals/GenericModal';
```

깊이를 많이 들락거리는 임포트 문에서는 절대경로로 변경하면 어떠신가요?